packages/app/src/hooks/useEraTimeLeft/index.tsx

```diff
@@ -25,8 +39,8 @@ export const useEraTimeLeft = () => {
     // Estimate the end time of the era
     const end = Number(start) + eraDuration
 
-    // Estimate remaining time of era
-    const timeleft = Math.max(0, end - getUnixTime(new Date()))
+    // Estimate remaining time of era using cached timestamp
```

Comment:
Because this is a getter fn, `new Date` is only executed when the consuming component calls it, so I don't think this caching will make a difference